pages/linux/ceph.md

```diff
@@ -0,0 +1,32 @@
+# ceph
+
+> Ceph implements object storage on a single distributed computer cluster, and provides interfaces for object, block and file-level storage.
```

Comment:
How about - "A tool provide object storage and maintenance on a distributed computer cluster" ?

Reply:
For more simplicity, it is better not to mention `distributed` and I rather say about the main responsibility of Ceph which is handling object storage. How about `Ceph uniquely delivers object, block, and file storage in one unified system.`?

Reply:
First off, we can remove "Ceph" from the description because it is mentioned in the title. Secondly, there are still too many words in the heading. Is there a way to combine "object, block and file" into a single thing ?

Reply:
For combining these words we can simply use `storage system` and my suggested description is `A unified storage system`. It's a general description of Ceph type tools. What about this?

Reply:
SGTM !